.github/workflows/bokeh-test-in-docker.yml

```diff
@@ -0,0 +1,46 @@
+name: Test in Docker
```

Comment:
Can we call this "Docker-CI" or "Bokeh-Docker", something more similar to the other existing workflow names?

Reply:
Switched to "Bokeh-Docker-Test" and changed the existing one to "Bokeh-Docker-Build" for consistency.